pkgs/applications/misc/openrgb/default.nix

```
@@ -1,35 +1,25 @@
-{ mkDerivation, lib, fetchFromGitHub, qmake, libusb1, hidapi, pkg-config, fetchpatch }:
+{ stdenv, lib, fetchFromGitLab, qt5, libusb1, hidapi, pkg-config }:
```

please follow https://nixos.org/manual/nixpkgs/stable/#sec-language-qt
```suggestion
{ mkDerivation, lib, fetchFromGitLab, qmake, libusb1, hidapi, pkg-config }:
```

---

Sorry, I used `pkgs.callPackage` and not `pkgs.libsForQt5.callPackage` in testing, fixed in rebase

---

to build this with a local checkout, all you need to do is:
```
nix-build -A openrgb
```